app/_src/gateway/production/running-kong/migrate-cassandra-to-postgres.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Migration guidelines: Cassandra to Postgres
+---
+
+The purpose of this document is to provide an approach aimed at migrating Cassandra db backed Kong deployment to Postgresql backed Kong deployment. This document will guide migration from traditional Kong deployment topology to hybrid.
+
+A Blue/Green migration approach is proposed in this document. To achieve the easiest possible migration, declarative tools Kong [DecK](https://docs.konghq.com/deck/latest/) will be utilized. DNS switching will be used to cut traffic over from a Blue environment to a Green environment. This will provide the ability to rollback faster if an issue is detected. 
+
+The steps to migrate, at a high level are: 
+
+* New target Kong environment will be built as a blank canvas with Postgresql db with hybrid deployment. 
+* Kong deck is used to take export (dump) from old traditional cassandra backed Kong deployment. 
+* Kong deck is used to import (sync) to new hybrid postgresql backed Kong deployment.
+* Basic auth credentials and local rbac users will need to be created separately either using admin api or manually(Kong Manager).
+* When the Kong configuration has been migrated over to the green environment, traffic can be slowly canaried into the green environment to de-risk the release with a fast rollback mechanism in place.
+
+
+### Considerations
+
+While the document provides high level guidelines, actual migration steps may differ based on the following factors:
+* complexity of the Kong deployment environment
+* Kong plugins used
+* Usage of custom plugins
+* External touchpoints to the Gateway (e.g. OIDC)
+* number of configuration entities - e.g. services and routes
+* If running an older version of Kong it is recommended to do a Kong version upgrade before the database migration (To reduce the moving parts in the upgrade procedure)
+* Using cassandra means may not be using the Kong Hybrid Mode deployment and might be deployed in the traditional mode, we recommend taking this opportunity to review the architecture to leverage the latest features of Kong Enterprise.
+
+While the diagram shows the split between the Kong control and Data planes, the same approach can be followed for Kong instances deployed in Classic Mode.
+
+![migration image](_assets/images/docs/gateway/migration.png)
+
+
+### Assumptions
+The Kong blue environment (Using Cassandra) and green environment (using Postgres) are running the same Kong Enterprise version
+Blue is deployed in traditional mode and green environment is running with Kong deployed in a hybrid deployment mode.
```

Comment:
```suggestion
## Prerequisites
* The {{site.base_gateway}} blue environment (using Cassandra) and green environment (using PostgreSQL) are running the same Gateway version.
* In the following examples, blue is deployed in traditional mode and green is deployed in hybrid mode. If you prefer to migrate to a traditional deployment, you can still follow the steps, but you don't need to worry about performing separate control plane and data plane tasks.
```
This guide should work for either OSS or Enterprise, correct?

Reply:
yes